packages/dd-trace/src/appsec/rasp/sql_injection.js

```diff
@@ -0,0 +1,86 @@
+'use strict'
+
+const { pgQueryStart, pgPoolQueryStart, wafRunFinished } = require('../channels')
+const { storage } = require('../../../../datadog-core')
+const addresses = require('../addresses')
+const waf = require('../waf')
+const { RULE_TYPES, handleResult } = require('./utils')
+
+const DB_SYSTEM_POSTGRES = 'postgresql'
+const reqQueryMap = new WeakMap() // WeakMap<Request, Set<querystring>>
```

Comment:
would `querytext` more appropriate? 
`querystring` can be confused with the http request query string

Reply:
makes sense, changing it.